### PR TITLE
Compatibility updates, fetching from private wiki

### PR DESCRIPTION
**Pull request by vedmaka**
- Removes composer autoloading in favor of `wfLoadExtension`. The extension will not load itself by default when installed via composer and instead will require the `wfLoadExtension` to be called explicitly
- Adds `ExternalRepositoryCredentials` option to allow querying from private wikis
- Adds REL1_35 compatibility changes
- Adds SMW 3.x compatibility changes
- Drops SMW 2.x support
- Drops MW < 1.28 support

### CODE REVIEW

File: extension.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+	"name": "SemanticExternalQueryLookup",
+	"version": "1.0.0-alpha",
+	"author": [
+		"James Hong Kong",
+		"Simon Heimler (gesinn.it GmbH & Co. KG)",
+		"Alexander Gesinn (gesinn.it GmbH & Co. KG)",
+		"Sebastian Schmid (gesinn.it GmbH & Co. KG)",
+		"Rcdeboer Remco C. de Boer"
```
Comment (JeroenDeDauw) [inline extension.json]: https://github.com/SemanticMediaWiki/SemanticExternalQueryLookup/graphs/contributors ?

Comment (alexander-gesinn) [reply extension.json]: The gesinn.it team definitely did not contribute. Looks like a copy & paste.

**Comment by kghbln** [comment]
Since the tests pass I will merge and release after the copy paste issue was corrected. Potential bug fixes can be done in consecutive release as need be.

File: extension.json (hunk 1 above)
Comment (vedmaka) [reply extension.json]: 🤔 I have no idea from where this did arrive from, fixed

**Pull request merged**